Customer.io project/components/Sidebar.tsx

```
@@ -0,0 +1,181 @@
+  const renderSection = (
+    title: string,
+    items: any[],
+    sectionKey: string,
+    icon: React.ReactNode
+  ) => (
+    <div className="mb-4">
+      <button
+        onClick={() => toggleSection(sectionKey)}
+        className="flex items-center justify-between w-full p-2 text-left hover:bg-gray-50 rounded-lg"
+      >
+        <div className="space-x-2 flex items-center justify-start">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)